content/md/supplier-onboarder/create-products.md

```diff
@@ -29,3 +29,22 @@ The family list will give you access to all the product families you can create
 :::info
 Product creation works the same way as in the PIM, learn more about it by reading our [Create a product](https://help.akeneo.com/articles/create-a-product.html#create-a-product) article.
 :::
+
+## Speed up with import
+Now, you know how to create a new product manually, you may think there should be a faster way to do it.
+
+The answer is yes!
+
+You can run an import and it's very easy.
+On your grid:
+1. Download the import (see [Download a product import template](dwl-product-import-tpl.html)),
+1. Complete the spreadsheet,
+1. On the product grid, click on the `Import` button,
+1. Drag and drop your file,
+1. Run the import and
+1. Wait for the notification
+
+
+::: warning
+Remember to specify a SKU for all your products to identify them.
```

Comment:
Not sure this warning is relevant. I'm not sure you can import a product without a SKU.

Reply:
removed